packages/components/src/markdown/hooks.jsx

```diff
@@ -0,0 +1,32 @@
+import { useCallback } from 'react';
+import { useHistory } from 'react-router';
+import { stripOriginFromUrl } from 'nebenan-helpers/lib/routes';
+import { invoke } from 'nebenan-helpers/lib/utils';
+
+const EMAIL_LINK_REGEX = /^mailto:/;
+const HTTP_LINK_REGEX = /^https?:/;
+const EXTENSION_REGEX = /\.\w+$/;
+
+export const useClickHandler = (onClick) => {
```

Comment:
imo would benefit from extracting somewhere - we use it in richcontent too

Reply:
Let's create a favro task for that issue. I am not super confident in extracting that part because it's used over multiple apps and they have slightly different implementations.

https://github.com/goodhood-eu/nebenan-react-hocs/blob/cf03f5bdb4750c7f4390d03b5b744d5ffd30fef0/src/use_smart_click_handler/index.jsx#L8
https://github.com/good-hood-gmbh/nebenan-frontend/blob/cce41446f4f5bc130ffcaa12eae117cde68f40fc/client/ui/rich_content/index.jsx#L34

Reply:
https://favro.com/organization/aee06c3fe6503c804f507dd9/22c630cc02b09f2d6f98b8b3?card=Goo-35767